config/application.yml.example

```diff
@@ -237,6 +237,7 @@ production:
   participate_in_dap: 'false' # pair with google_analytics_key
   password_pepper: # generate via `rake secret`
   password_strength_enabled: 'true'
+  piv_cac_agencies: 'DOD'
```

Comment:
Would it help to make this an array from the outset? (Looks like it's intended as a delimited string for now, though maybe there won't be many of these coming along.)

Reply:
It is intended to be a comma-delimited string. Figaro requires strings. Rather than parse as JSON, I just split on commas.